dGame/dUtilities/VanityUtilities.cpp

```diff
@@ -36,19 +45,17 @@ void VanityUtilities::SpawnVanity() {
 			info.pos = { 259.5f, 246.4f, -705.2f };
 			info.rot = { 0.0f, 0.0f, 1.0f, 0.0f };
 			info.spawnerID = Game::entityManager->GetZoneControlEntity()->GetObjectID();
-
-			info.settings = { new LDFData<bool>(u"hasCustomText", true),
-				new LDFData<std::string>(u"customText", ParseMarkdown((BinaryPathFinder::GetBinaryDir() / "vanity/TESTAMENT.md").string())) };
+			info.settings = {
+				new LDFData<bool>(u"hasCustomText", true),
```

Comment:
A bit confused by the heap allocations here. Doesn't seem _wrong_, I just don't get it.

Reply:
If there's a better way to do it, lmk. That's just how it is.

Reply:
Not an issue with this PR even if it is an issue (which I'm not sure of). If I have time later, I'll look into it myself, but I'll spin up a new PR in that case. Was just kind of hoping you had an easy explanation for me. 😅

Reply:
Thr heap allocation is necessary as ldf in dlu is designed poorly and hard to discern ownership.